### DISCUSSION TITLE
Share point / OneDrive web interface macOS

### DISCUSSION THREAD

**Discussion by aurelienb1** [SafeExamBrowser/seb-mac#127] [Q&A]
Hi,

We explore another to make open book exam. The solution is based on seb and sharepoint/onedrive.

In SEB windows it works correctly, I manage to open sharepoint online and to be able to consult the documents.

However under mac, the O365 online interface does not even appear, see screenshot.

![MicrosoftTeams-image](https://user-images.githubusercontent.com/51001741/132875743-886968c0-cd02-4cf0-a7cb-60e63ee98b69.png)

**Comment by danschlet** [comment]
Most likely O365 isn't compatible with the classic WebKit browser engine. Try it with the [release candidate of SEB 3.0](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.0rc), which uses the modern WebKit browser engine. Please note the restrictions (in case you use URL filters with the option "Filter also embedded content" enabled, the classic WebView will be used in SEB 3.0).
